README.md

```diff
@@ -0,0 +1,28 @@
+# Student alcohol consumption
+
+## Dataset
+[Student Alcohol Consumption](https://www.kaggle.com/datasets/uciml/student-alcohol-consumption?select=student-mat.csv)
+
+This dataset contains information about students that may influence their academic performance.
+
+## The purpose of the prediction 
+To determine the student's final grades based on their alcohol consumption and other characteristics (listed below) using various models and our own linear regression model
+
+## Selected features for prediction
```

Comment:
Add all features from https://github.com/PNUAITeam13/AlcoholConsumption/blob/main/data_attributes.txt

Reply:
And maybe use ```-``` instead of ```+```
- **age**: Student's age (numeric: from 15 to 22)